### DISCUSSION TITLE
ejs diagram library not implementing

### DISCUSSION THREAD

**Discussion by chhabumanisha** [npm/feedback#483] [CLI] [answered]
'ejs-diagram' is not a known element:\n1.
 If 'ejs-diagram' is an Angular component, then verify that it is part of this module.\n2. If 'ejs-diagram' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message."

**Comment by MylesBorins** [comment, marked as answer]
Hey @chhabumanisha . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
